alpha/veneer/basic/basic.go

```diff
@@ -67,7 +43,8 @@ func (v Veneer) renderFile(ctx context.Context, ref string) (*declcfg.Declarativ
 	return cfg, nil
 }
 
-// isBundleVeneer identifies loaded partial Bundle data from YAML/JSON veneer source as having no properties,
+// isBundleVeneer identifies a Bundle veneer source as having a Schema and Image defined
+// but no Properties, RelatedImages or Package defined
 func isBundleVeneer(b *declcfg.Bundle) bool {
-	return len(b.Properties) == 0
+	return b.Schema != "" && b.Image != "" && b.Package == "" && len(b.Properties) == 0 && len(b.RelatedImages) == 0
```

Comment:
We should have the field restrictions for the simple veneer documented, specifically the hard requirement for prohibited fields. The error message isn't very helpful here.